test/format/failure_configs/CONFIG.WT-11710

```diff
@@ -37,7 +37,6 @@ debug.slow_checkpoint=0
 debug.table_logging=0
 debug.update_restore_evict=1
 disk.data_extend=1
-disk.direct_io=1
```

Comment:
As discussed IRL - would be good to have an explanation somewhere (on the PR or the ticket) for why we found it acceptable to remove this.

Also - wasn't there a second format config with this set? Or am I mis-remembering.

Reply:
Good spot - I had a few more config removals I forgot to push the commit for!